### ISSUE TITLE
Police Oversight form: Accessibility - form labels

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1158]
Several areas where missing/orphaned form labels are causing accessibility errors: 
- [ ] Description (missing)
- [ ] Date and time (missing)
- [ ] Map (missing)
- [ ] File upload (orphaned)

![Screen Shot 2019-02-05 at 6.52.11 PM.png](https://images.zenhubusercontent.com/5a5f8dc94b5806bc2bc322a5/c4eb3514-a08e-40aa-aa3c-b46d83cb439e)

<img width="796" alt="screen shot 2019-02-23 at 8 49 57 pm" src="https://user-images.githubusercontent.com/1463708/53294469-5a938d00-37ad-11e9-9d20-04acc4495518.png">

From WAVE tool: 
> **How to Fix It**
If a text label for a form control is visible, use the `<label>` element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using `aria-labelledby`. Labels are not required for image, submit, reset, button, or hidden form controls.
**The Algorithm... in English**
An `<input>` (except types of image, submit, reset, button, or hidden), `<select>`, or `<textarea>` does not have a properly associated label text. A properly associated label is:
a `<label>` element with a for attribute value that is equal to the id of a unique form control
a `<label>` element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute
a non-empty title attribute, or a non-empty` aria-labelledby` attribute.

>

**Comment by easherma** [comment]
@sergiogcx @briaguya would love to discuss when we're in person next, we may need to tweak our file upload widget and other widgets to pass IDs

https://github.com/usds/us-forms-system/blob/master/src/js/widgets/TextWidget.jsx
https://github.com/usds/us-forms-system/blob/master/docs/customizing-the-library/creating-custom-fields-and-widgets.md

**Comment by easherma** [comment]
I can't quite figure this out so far. 
Mozilla's json schema has some documentation on ids being generated. 
us-forms customized some of this. 
It seems clear enough that the label id is being generated, but not associated with the component. 
It should be a simple thing to fix this, it would be painless with html. 
I haven't figured out a good approach with React/between these two libraries.

**Comment by easherma** [comment]
@amenity if this is a priority before press push, I think may need to pair with @briaguya  or @sergiogcx

**Comment by Niicck** [comment]
I'm working on aria stuff for date and time.

**Comment by Niicck** [comment]
Blocked by https://github.com/cityofaustin/techstack/issues/1548 for 100% WAVE compliant accessibility.
